services/app/src/contexts/ChessBoardContext.jsx

```diff
@@ -179,7 +190,15 @@ const ChessBoardContextProvider = ({ children }) => {
       if (voters.length === 1) {
         setGlobalScore(0)
       } else {
-        const scoresArray = turns.map(turn => turn.score);
+        const scoresArray = turns
+          .filter(turn => turn.turnType === PlayerStatuses.FigurePlaced)
+          .map(turn => turn.score);
+        
+        if (scoresArray.length === 0) {
```

Comment:
Lest make calculate average to return 0 in case empty array is passed. It will avoid these ifs